src/ast/mod.rs

```diff
@@ -2516,6 +2516,35 @@ pub enum Statement {
     /// CREATE TABLE
     /// ```
     CreateTable(CreateTable),
+    /// ``` sql
+    /// CREATE ICEBERG TABLE
+    /// Snowflake-specific statement
+    /// <https://docs.snowflake.com/en/sql-reference/sql/create-iceberg-table>
+    /// ```
+    CreateIcebergTable {
```

Comment:
The preferred path would be to reuse the `CreateTable` variant, by e.g. adding a `is_iceberg_table: bool` to flag the table type and then including any iceberg table fields as optional/empty

Reply:
Ok, sure. I just noticed that `CreateVirtualTable` for SQLLite is a separate statement.